tests-web/assets/js/tests.js

```diff
@@ -18,6 +18,8 @@ export const makeOpenTest = (map) => async (name) => {
     const networkLayer = await makeDotLayer(await network, { bounds })
     map.addLayer(networkLayer);
 
+    L.control.layers({}, {"Geometry": rawMapLayer, "OSM input": inputLayer, "Graph": networkLayer}).addTo(map);
```

Comment:
What're we using for formatting? I used `prettier`, but it changed a bunch of stuff here. VS code had a slight change in the imports. I'm happy to match whatever

Reply:
Prettier is my preferred formatter (as long as trailing commas is turned on). The existing code does not match the `prettier` format, but I am in favour of changing over to it.

For now, format it however looks decent, and I will set up prettier with a config and reformat everything soon.